src/components/OrgFile/index.js

```diff
@@ -114,11 +114,16 @@ class OrgFile extends PureComponent {
     }
   }
 
-  componentDidUpdate() {
+  componentDidUpdate(prevProps) {
     const { headers, pendingCapture } = this.props;
     if (!!pendingCapture && !!headers && headers.size > 0) {
       this.props.org.insertPendingCapture();
     }
+
+    const { path } = this.props;
```

Comment:
If we don't really need `path`/`dir`, we could revert this hunk.

Reply:
No we can't.  This is essential for making `<Link>` work in this context, otherwise the address bar changes but the displayed file doesn't.